account_chatter/models/account_account.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Vauxoo
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models, fields
+
+
+class AccountAccount(models.Model):
+
+    _name = "account.account"
+    _inherit = ['account.account', 'mail.thread']
+
+    name = fields.Char(tracking=True)
+    currency_id = fields.Many2one(tracking=True)
+    code = fields.Char(tracking=True)
+    deprecated = fields.Boolean(tracking=True)
+    reconcile = fields.Boolean(tracking=True)
+    user_type_id = fields.Many2one(tracking=True)
+    tax_ids = fields.Many2many(tracking=True)
+    tag_ids = fields.Many2many(tracking=True)
+    group_id = fields.Many2one(tracking=True)
+
+    def _mail_track(self, tracked_fields, initial):
+        """ For a given record, fields to check (tuple column name, column info)
+        and initial values, return a valid command to create tracking values.
+        :param tracked_fields: fields_get of updated fields on which tracking
+          is checked and performed;
+        :param initial: dict of initial values for each updated fields;
+        :return: a tuple (changes, tracking_value_ids) where
+          changes: set of updated column names;
+          tracking_value_ids: a list of ORM (0, 0, values) commands to create
+          ``mail.tracking.value`` records;
+        Override this method on a specific model to implement model-specific
+        behavior. Also consider inheriting from ``mail.thread``. """
+        self.ensure_one()
+        changes = set()  # contains onchange tracked fields that changed
+        tracking_value_ids = []
+
+        # generate tracked_values data structure: {'col_name': {col_info, new_value, old_value}}
+        for col_name, col_info in tracked_fields.items():
+            if col_name not in initial:
+                continue
+            initial_value = initial[col_name]
+            new_value = self[col_name]
+
+            if new_value != initial_value and (new_value or initial_value):  # because browse null != False
+                tracking_sequence = getattr(
+                    # backward compatibility with old parameter name
+                    self._fields[col_name], 'tracking', getattr(self._fields[col_name], 'track_sequence', 100))
+                if tracking_sequence is True:
+                    tracking_sequence = 100
+                tracking = self.env['mail.tracking.value'].create_tracking_values(
+                    initial_value, new_value, col_name, col_info, tracking_sequence, self._name)
+                if tracking:
+                    if tracking['field_type'] == 'monetary':
```

Comment:
Not necessary, as this inheritance is to add support to many2many fields

Reply:
@luisg123v could you improve your comment, please ?

Reply:
This is in the original method because it's intended to support several field types, including monetary fields. Since you're extending that method to add value to a many2many field, this line is not required, because it's only applicable to monetary fields

Reply:
Done